src/docs/Contributing/Principles/ContributionJourney/index.mdx

```diff
@@ -0,0 +1,120 @@
+
+We identified **6 phases** in this contribution funnel. Let's briefly explain what every phase is about.
+
+# 1. KICK-OFF
+
+### ⁉️ What ?
+The kick-off is the time where the designer and the developer sit alongside with Mozaic and LEROY MERLIN's teams. This time is very crucial because both teams will explain the duo what's expected from them. We like to do this event **in real life**.
+
+### 👩‍🚀 Who ?
+- Mozaic's team
+- LMF Core team
+- The designer and developer duo
+
+### 🎬 How ?
+The duo has to ask himself, Where does this pattern go? What does the duo want from this pattern: the duo needs to do a pooling on paper for the design of the pattern and have a small talk on what they want to achieve with the documentation. This will allow them to speak about technical feasibility.
+
+### 🚀 Deliverables
+
+[ ]  Paper sketching of the pattern
+[ ]  Documentation draft on a shared Google Doc
+
+# 2. DESIGN
+
+### ❓ What
+
+The design phase is a time consuming part of the contribution. For that reason, the designer will have to work on its own for a little time to do the research it's needed. The developer doesn't need to be available during this period other than just answering a few questions from the designer if he has some.
+
+<Highlight theme="tips" title="">
+
+💡 The developer can take advantage of this period to install the project on his environment and try to understand how the files and the documentation is structured. Feel free to contact Mozaic's tech team on Slack if you need any kind of support.
+
+</Highlight>
+
+### 👩‍🚀 Who ?
+
+- Designer
+- Mozaic's UI support
+
+### 🎬 How ?
+
+The designer working on this particular pattern will definitely need to pair with a Mozaic's UI.
+
+### ⏲ How long ?
+
+This task can take up to a week to be done well for some complex patterns.
+
+### 🚀 Deliverables
+
+- [ ]  First version of the Sketch files
+
+# 3. FIRST POOLING
+
+### ❓ What ?
+
+This part is very important because this is when the designer will submit his work to the developer. The developer will evaluate if the design work is technically doable. The other part of the pooling will allow them to talk about the first documentation draft. Is it still relevant and does it mean something for the end user ?
+
+### 👩‍🚀 Who ?
+
+The duo will also have to see if the designed pattern responds to digital standards (accessibility, UX behaviour, etc.). They can ask for Mozaic's team and LMF core teams.
+
+### 🎬 How ?
+
+The duo will be exchanging together to discuss technical and design issues **together**.
+
+### 🚀 Deliverables
+
+- [ ]  Technical validation
+- [ ]  Documentation summary validated
+- [ ]  For the designer only, **Abstract** collections with:
+    - [ ]  Spacings specs
+    - [ ]  Board with different variations (sizes, cases, etc.)
+    - [ ]  Board with different states
```

Comment:
For the designer only, **Abstract** collections with:
    - Spacings specs
    - Board with different variations (sizes, cases, states, etc.)